client/test/preview/integration/route/digitaltwins/manage/DeleteDialog.test.tsx

```diff
@@ -8,7 +8,7 @@ import digitalTwinReducer, {
 } from 'preview/store/digitalTwin.slice';
 import snackbarSlice from 'preview/store/snackbar.slice';
 import DigitalTwin from 'preview/util/digitalTwin';
-import { mockBackendInstance } from 'test/preview/__mocks__/global_mocks';
+import { mockBackendInstance } from 'test/__mocks__/global_mocks';
```

Comment:
This is a good candidate for global mocks because it's code is outside of `preview`.